android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java

```diff
@@ -37,9 +43,15 @@ public void getAll(Callback callback) {
     ContentResolver cr = getReactApplicationContext().getContentResolver();
     Uri uri = ContactsContract.Contacts.CONTENT_URI;
     String selection = ContactsContract.Contacts.IN_VISIBLE_GROUP + " = " + 1;
+    Resources resources = this.reactContext.getResources();
     String sortBy = CommonDataKinds.Contactables.LOOKUP_KEY;
-
-    Cursor cursor = cr.query(uri, null, selection, null, sortBy);
+    String [] projection = new String[]{
+            CommonDataKinds.Contactables._ID,
+            CommonDataKinds.Contactables.DISPLAY_NAME,
```

Comment:
turns out i cant just add them, it would require a different query n some more thought.. may do it in another changeset